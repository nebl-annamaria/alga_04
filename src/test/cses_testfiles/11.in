1000 3
A#B
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
.#.
...
