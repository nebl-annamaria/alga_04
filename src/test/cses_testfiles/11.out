YES
2000
DDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDRRUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU
